pages/manage_and_operate/kms/kms-troubleshooting/meta.yaml

```
@@ -0,0 +1,3 @@
+id: 751e237a-47e4-4ac8-854d-189530462197
+full_slug: kms-troubleshooting
```

```suggestion
full_slug: kms-logs
```
@gbarideau , after reading the guide IMO "logs" would be more convenient

---

Yes, agreed